modules/openapi-generator/src/main/resources/typescript-nestjs/api.service.mustache

```diff
@@ -8,7 +8,7 @@ import { HttpService } from '@nestjs/axios';
 {{^useAxiosHttpModule}}
 import { HttpService, Injectable, Optional } from '@nestjs/common';
 {{/useAxiosHttpModule}}
-import { AxiosResponse } from 'axios';
+import { AxiosRequestConfig, AxiosResponse } from 'axios';
```

Comment:
I just noticed: should this be be a `type` only import if we assume TS > 3.8?


refs https://github.com/OpenAPITools/openapi-generator/pull/20064